blackboxopt/optimizers/staged/bohb.py

```diff
@@ -285,6 +338,7 @@ def sample_configuration(self) -> Tuple[dict, dict]:
                 )
 
         except Exception:
+            raise
```

Comment:
As far as I can see, the `raise` here results in the log output and return statement below are not executed.
If in this case they aren't needed anymore, please remove them. And in case they are needed. please remove the raise statement.

Reply:
Sorry, I missed that. I just introduced that there to see if something is failing in the model based sampling. I removed it again.